src/transformers/tokenization_utils.py

```diff
@@ -190,6 +195,10 @@ def split(self, text: str) -> List[str]:
                         # End lookahead
 
                     # Storing and resetting
+                    if offsets and start < offsets[-1]:
+                        import ipdb
+
+                        ipdb.set_trace()
```

Comment:
I don't think this is supposed to be here.

Reply:
No it wasn't. :)